packages/core/src/components.d.ts

```diff
@@ -101,6 +103,18 @@ export namespace Components {
         "removable"?: boolean;
         "status"?: 'primary' | 'info' | 'success' | 'warning' | 'danger';
     }
+    interface DsoMapBaseLayers {
+        "baseLayers": BaseLayer[];
+        "selectedBaseLayer": BaseLayer | undefined;
+    }
+    interface DsoMapControls {
+        "disableZoom"?: 'in' | 'out' | 'both';
+        "open": boolean;
+    }
+    interface DsoMapOverlays {
```

Comment:
Overweging: De web components `<dso-base-layers>` en `<dso-overlays>` ombouwen tot Functional Components en aansturing uitsluitend via props op `<dso-map-controls>`

Reply:
In de toekomst hebben we wellich de flexibiliteit van losse Web Components nodig. Dus ondanks dat ik het mooier vind als het gefuseerd wordt tot 1 Web Component laat ik het en heb ik de README.md files van map-base-layers en map-overlays verrijkt met de melding dat het een private API is